src/ca65/main.c

```diff
@@ -129,7 +130,8 @@ static void Usage (void)
             "  --smart\t\t\tEnable smart mode\n"
             "  --target sys\t\t\tSet the target system\n"
             "  --verbose\t\t\tIncrease verbosity\n"
-            "  --version\t\t\tPrint the assembler version\n",
+            "  --version\t\t\tPrint the assembler version\n"
+            "  --segment-list\t\tEnable segment offset listing\n",
```

Comment:
The new line should go after "--relax-checks" (options are alphabetically sorted)